Shorten sentences (#3561)
viamrobotics/docs: docs/_index.md

<div class="explanation">
  <div class="explanationtext">

> Viam has a built-in tool called _{{< glossary_tooltip term_id="fragment" text="fragments" >}}_ for managing hardware and software for multiple machines.

Author: ```suggestion
Manage hardware and software for multiple machines using a built-in tool called _{{< glossary_tooltip term_id="fragment" text="fragments" >}}_.
```

<div class="explanation">
  <div class="explanationtext">

Provisioning allows you to complete part of the machine setup during the manufacturing process. The rest of the first-time setup happens once the machine is taken into operation.

> This way machines will automatically get the latest updates.

Author: ```suggestion
This way, machines automatically get the latest updates.
```

<div class="explanation">
  <div class="explanationtext">

> Build machine learning models based on your machines' data. You can use builtin training algorithms or your own.

Author: ```suggestion
Build machine learning models based on your machines' data. You can pick from available training algorithms or create your own.
```

> Build machine learning models based on your machines' data. You can pick from available training algorithms or create your own.

Author: ```suggestion
Build machine learning models based on your machines' data. You can pick from different training algorithms or create your own.
```

<div class="explanation">
  <div class="explanationtext">

> Viam allows you to organize and manage any number of machines. When collaborating with others you can assign permissions using Role-Based Access Control (RBAC).

Author: ```suggestion
Viam allows you to organize and manage any number of machines. When collaborating with others, you can assign permissions using Role-Based Access Control (RBAC).
```
